src/greentea/gtea/greentea_log.py

```diff
@@ -14,7 +14,7 @@
 
 
 class GreenTeaSimpleLockLogger(object):
-    """Simple locking printing mechanism."""
+    """Simple lock printing mechanism."""
```

Comment:
What's a lock printing mechanism? Do we mean a thread-safe printing mechanism? I assume we're trying to guarantee serialised output in a multi-threaded context with this object.